RetailCoder.VBE/Inspections/InspectionBase.cs

```diff
@@ -81,6 +83,28 @@ protected virtual IEnumerable<Declaration> BuiltInDeclarations
             get { return State.AllDeclarations.Where(declaration => declaration.IsBuiltIn); }
         }
 
+        protected bool HasIgnoreAnnotation(VBComponent component, int line)
```

Comment:
I could move them both into InspectionBase and make them overloads, yes, but I don't think I can do much better than that.